Controllers/BaseController.php

```diff
@@ -212,4 +212,46 @@ protected function getNameFromUuid(string $table, string $uuid): string
         }
         return '';
     }
+
+    /**
+     * Function for generating REST response
+     *
+     * @return void
+     */
+    public function apiData(): void
+    {
+        //TODO: adding authentication of the user, so only certain users can access this API
+
+        $table = $this->getFilenameFromClass();
+        $data = Storage::loadElements($table);
+
+        if (empty($data)) {
+            $this->sendResponse(500, [
+                'status' => 'error',
+                'message' => 'Failed to load data from storage!',
+                'data' => []
+            ]);
+        }
+
+        $this->sendResponse(200, [
+            'status' => 'success',
+            'message' => 'Data loaded successfully!',
+            'data' => $data
+        ]);
+    }
+
+    /**
+     * function for sending API response with specific status code and data
+     *
+     * @param int $statusCode
+     * @param array $data
+     * @return void
+     */
+    protected function sendResponse(int $statusCode, array $data): void
+    {
+        http_response_code($statusCode);
```

Comment:
You can use your `json` helper method

Reply:
Json helper method will be used here once merged.